conf/machine/imx6qdlsabresd.conf

```diff
@@ -41,18 +41,25 @@ KERNEL_DEVICETREE_use-mainline-bsp = " \
     imx6dl-sabresd.dtb \
 "
 
-UBOOT_MACHINE ?= "mx6sabresd_defconfig"
+### u-boot-fslc settings ###
 
-# These u-boot variables default to the correct settings for u-boot-imx.
-# Set overrides for u-boot-fslc.
-UBOOT_MAKE_TARGET_pn-u-boot-fslc = "all"
-UBOOT_SUFFIX_pn-u-boot-fslc = "img"
 SPL_BINARY_pn-u-boot-fslc = "SPL"
-WKS_FILE_pn-u-boot-fslc = "imx-uboot-spl-bootpart.wks.in"
+UBOOT_MACHINE_pn-u-boot-fslc ?= "mx6sabresd_defconfig"
+UBOOT_SUFFIX_pn-u-boot-fslc = "img"
```

Comment:
Should the ?= assignment be consistent for all these variables?

Reply:
Good question. Do we want it to be overridable in `local.conf`?